cn.apipost.apipost.json

```diff
@@ -0,0 +1,133 @@
+        },
+        {
+            "name": "noto-font-cjk-bold",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/fonts",
+                "install -D NotoSansCJK-Bold.ttc /app/share/fonts/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJK-Bold.ttc.zip",
+                    "sha256": "3cd895bce6765903b2eefcd90eea6ed372364e66d5b896ea3a594bd3e737eba0"
+                }
+            ]
+        },
+        {
+            "name": "noto-font-cjk-medium",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/fonts",
+                "install -D NotoSansCJK-Medium.ttc /app/share/fonts/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJK-Medium.ttc.zip",
+                    "sha256": "d7590637168e1cff4fd3a15422e483308d7ba2a7120ccaf96ead274c3b40e1b3"
+                }
+            ]
+        },
+        {
+            "name": "apipost",
+            "builddir": true,
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 cn.apipost.apipost.metainfo.xml -t $FLATPAK_DEST/share/metainfo",
+                "install -Dm644 cn.apipost.apipost.desktop /app/share/applications/cn.apipost.apipost.desktop",
+                "install -Dm644 cn.apipost.apipost.svg /app/share/icons/hicolor/scalable/apps/cn.apipost.apipost.svg",
+                "install -Dm755 run.sh /app/bin/run.sh",
+                "install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra"
+
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "cn.apipost.apipost.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "cn.apipost.apipost.metainfo.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "cn.apipost.apipost.svg"
+                },
+                {
+                    "type": "file",
+                    "path": "run.sh"
+                },
+                {
+                    "type": "extra-data",
+                    "filename": "apipost_linux.tar.xz",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://www.apipost.cn/dl.php?client=LinuxXz&arch=x64&version=8.1.4",
+                    "sha256": "f5d394a00e5ded23b9834b4322ff8e5381dc3643fb4a4f0a1c43ea2d44f28294",
+                    "size": 74828724
+                },
+                {
+                    "type": "script",
+                    "commands": [
+                        "tar xvf apipost_linux.tar.xz",
```

Comment:
you have to silence tar output here otherwise it leaks in when installing

Reply:
https://github.com/flathub/flathub/pull/5735/commits/d4cca54f93365868d70fb3a6f87363b4ee2501c3